StandardLibrary/src/String.dfy

```diff
@@ -48,4 +50,26 @@ module StandardLibrary.String {
   {
     Int2String(n, Base10)
   }
+
+  /* Returns the index of a substraing or None, if the substring is not in the string */
+  method HasSubString(xs: string, ss: string)
+    returns (o: Wrappers.Option<nat>)
+    requires |ss| <= |xs|
```

Comment:
It might be more useful to skip this requirement, and just return None in this case.
Otherwise, I just have to write the `if |ss| <= |xs|`everywhere I use it.

Reply:
indeed, the code doesn't even need to change (I think)!